docs/application/dotnet/guides/nui/Palette.md

```diff
@@ -0,0 +1,102 @@
+# Palette
+
+The palette APIs is a support that extracts prominent colors from images to help you create visually engaging apps.
+
+You can use the palette APIs to design layout themes and apply custom colors to visual elements in your app.
+For example, you can use a palette to create a color-coordinated title card for a song based on its album cover or to adjust an app’s toolbar color when its background image changes.
+The Palette object gives you access to the colors in a Bitmap image while also providing six main color profiles from the bitmap to help inform your design choices.
+
+## Add namespace
+To implement Palette, include `Tizen.NUI` namespace in your application:
+
+```csharp
+using Tizen.NUI;
+```
+
+## Create a palette
+
+Generate a Palette instance using PixelBuffer method to first create a PixelBuffer from a Image.
+
+1. Create PixelBuffer from source image:
+
+    ```csharp
+    PixelBuffer imgBitmap = ImageLoading.LoadImageFromFile(imagePath);   
+    ```
+then generate the palette either synchronously or asynchronously.
+
+2. Generate a Palette synchronously or asynchronously:
+    ```csharp
+    // Generate a Palette synchronously
+    public void PaletteGeneratesync()
+    {
+        PixelBuffer imgBitmap = ImageLoading.LoadImageFromFile(_imgPath);
+        try
+        {
+            Palette palette = Palette.Generate(imgBitmap);
+        }
+        catch (ArgumentNullException e)
+        {
+            //Exception Handling.
+        }
+    }
+    
+    // Generate a Palette asynchronously
+    public async Task PaletteGenerateAsync()
+    {
+        PixelBuffer imgBitmap = ImageLoading.LoadImageFromFile(_imgPath);
+        try
+        {
+            Palette palette = await Palette.GenerateAsync(imgBitmap);
+        }
+        catch (ArgumentNullException e)
+        {
+            //Exception Handling.
+        }
+    }
+    ```
+ 
+ 3. Set a region of the pixelBuffer to be used exclusively when calculating the palette. (Optional)
+ 
+    ```csharp
+    Rectangle rect = new Rectangle(0, 0, 100, 100);
+    ...
+    Palette palette = Palette.Generate(imgBitmap, rect);
+    ```
+
+## Extract color profiles
+
+Colors extracted from the bitmap image are scored against each profile based on saturation, luminance, and population (number of pixels in the bitmap represented by the color).
+For each profile, the color with the best score defines that color profile for the given image.
+Palette object contains 16 primary colors from a given image.
+
+The palette library attempts to extract the following six color profiles:
```

Comment:
Doubt ? Attempts to extract or it extracts colors? 

Is this fine? check whether it is technically fine. 

The palette library extracts the following six color profiles:

Reply:
Yes.

Attempts to create 6 color profiles, and if no matching color is extracted from the image, it will not be created.

So I used the expression to attempt.

Reply:
```suggestion
The Palette library attempts to extract the following six color profiles: